doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -84,7 +84,7 @@ comment_on ::=
   'COMMENT ON' (
     'CLUSTER' | 'CLUSTER REPLICA' | 'COLUMN' | 'CONNECTION' | 'DATABASE' | 'FUNCTION' |
     'INDEX' | 'MATERIALIZED VIEW' | 'ROLE' | 'SCHEMA' | 'SECRET' | 'SINK' | 'SOURCE' |
-    'TABLE' | 'TYPE' | 'VIEW'
+    'TABLE' | 'TYPE' | 'VIEW' | 'NETWORK POLICY'
```

Comment:
you need to regenerate the railroad diagrams when making a change here, see [these docs](https://github.com/MaterializeInc/materialize/tree/main/doc/user#to-generate-diagrams)

Reply:
thanks for the reminder